### PR TITLE
Update error page layout

### LINKED ISSUE #220: Error pages are confusing on mobile
_...especially for non-native English readers_

As they say `[error] but there's nothing if you are on a mobile device`. It's time to probably sunset Pong and just give a small and concise message + pointers.

### PR DESCRIPTION
**Pull request by pixeldesu**
Fixes #220

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -1,56 +1,139 @@
...
+      <img src="/images/errors/small/404.png"/>
+    </picture>
+  </div>
+  <div class="text-center">
+    <h1>
+      <span class="text-primary font-weight-bold">404</span>
+      <small class="text-muted">Page not found</small>
+    </h1>
+    <p>
+      Whatever you might have been searching for, it's not here!
```
Comment (nilsding) [inline public/404.html]: reword?

```suggestion
      Whatever you might have been looking for, it's not here.
```

File: public/422.html [hunk 2]
```diff
@@ -1,58 +1,150 @@
...
+      <img src="/images/errors/small/404.png"/>
+    </picture>
+  </div>
+  <div class="text-center">
+    <h1>
+      <span class="text-primary font-weight-bold">422</span>
+      <small class="text-muted">Unprocessable Entity</small>
+    </h1>
+    <p>
+      Whatever your browser tried to send us, it was not right!
```
Comment (nilsding) [inline public/422.html]:
```suggestion
      Whatever your browser tried to send us, it was not right.
```

File: public/422.html [hunk 3]
```diff
@@ -1,58 +1,150 @@
...
+    </p>
+    <p class="mb-0">
+      <i>Please retry what you initially tried, and if the problem persists, don't hesitate to contact us!</i>
```
Comment (nilsding) [inline public/422.html]:
```suggestion
      <i>Please retry what you just did.  If the problem persists, don't hesitate to contact us:</i>
```

File: public/422.html [hunk 4]
```diff
@@ -1,58 +1,150 @@
...
+    </p>
+    <ul class="links mt-0">
+      <li>
+        <a href="https://twitter.com/retrospring">Twitter</a>
```
Comment (nilsding) [inline public/422.html]: might make sense to include the handle in the link text so people can immediately @ us from their preferred app?

```suggestion
        <a href="https://twitter.com/Retrospring">Twitter: @Retrospring</a>
```

File: public/422.html [hunk 5]
```diff
@@ -1,58 +1,150 @@
...
+      </li>
+      <li>
+        <a href="https://github.com/retrospring/retrospring">GitHub</a>
```
Comment (nilsding) [inline public/422.html]:
```suggestion
        <a href="https://github.com/Retrospring/retrospring">GitHub</a>
```

File: public/500.html [hunk 6]
```diff
@@ -1,60 +1,150 @@
...
+      <img src="/images/errors/small/500.png"/>
+    </picture>
+  </div>
+  <div class="text-center">
+    <h1>
+      <span class="text-primary font-weight-bold">500</span>
+      <small class="text-muted">Internal Server Error</small>
+    </h1>
+    <p>
+      If this page shows up, something went wrong and we already sent out our army of raccoons to fix it!
```
Comment (nilsding) [inline public/500.html]:
```suggestion
      If this page shows up, something went wrong.  We dispatched our army of raccoons to fix it.
```

File: public/500.html [hunk 7]
```diff
@@ -1,60 +1,150 @@
...
+    </p>
+    <p class="mb-0">
+      <i>We get notified about issues, but feel free to contact us:</i>
+    </p>
+    <ul class="links mt-0">
+      <li>
+        <a href="https://twitter.com/retrospring">Twitter</a>
```
Comment (nilsding) [inline public/500.html]:
```suggestion
        <a href="https://twitter.com/Retrospring">Twitter: @Retrospring</a>
```

File: public/500.html [hunk 8]
```diff
@@ -1,60 +1,150 @@
...
+      </li>
+      <li>
+        <a href="https://github.com/retrospring/retrospring">GitHub</a>
```
Comment (nilsding) [inline public/500.html]:
```suggestion
        <a href="https://github.com/Retrospring/retrospring">GitHub</a>
```

**Review by nilsding — COMMENTED** [review]
I've got some suggestions.  will test these pages on everything I have

**Review by raccube — COMMENTED** [review]
Too big on 13" MacBook Pro at default display scaling, same on iPad Pro in Landscape
Screenshot 2022-01-01 at 17 36 47 (2)

File: public/502.html [hunk 9]
```diff
@@ -74,7 +74,7 @@
   <div class="dialog">
     <div>
       <h1>This page takes way too long to load!</h1>
-      <img src="images/angry_unicorn.png" alt="Unicorn">
+      <img src="images/errors/original/angry_unicorn.png" alt="Unicorn">
```
Comment (nilsding) [inline public/502.html:77]: Funny how we still have the unicorn, even though we switched to Puma ... 😜

**Comment by nilsding** [comment]
can confirm that it's too big on landscape.  Also, viewing them side by side makes it obvious that they're different sized

ideally there should be no scrollbar here, maybe on landscape the raccoon can be on the left side?

| Screen Shot 2022-01-01 at 17 51 49 | Screen Shot 2022-01-01 at 17 51 33 |
|-|-|

File: public/404.html [hunk 10]
```diff
@@ -1,56 +1,139 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Page not found!!1! (404)</title>
+  <meta charset="utf-8">
+  <title>Page not found!</title>
```
Comment (nilsding) [inline public/404.html]:
```suggestion
  <title>Page not found</title>
```

**Comment by pixeldesu** [comment]
Error pages now use horizontal space more when available and wrap on smaller screen sizes.

File: public/404.html [hunk 11]
```diff
@@ -1,56 +1,151 @@
...
+  <div class="error">
+    <div class="error-artwork-container">
+      <picture>
+        <source
+          srcset="/images/errors/small/404.png 576w,
+                  /images/errors/medium/404.png 768w,
+                  /images/errors/original/404.png 1200w">
+        <img src="/images/errors/small/404.png"/>
```
Comment (nilsding) [inline public/404.html]: For accessibility:
```suggestion
        <img src="/images/errors/small/404.png" alt="A raccoon peeking out of a bin, looking confused" />
```

File: public/422.html [hunk 12]
```diff
@@ -1,58 +1,162 @@
...
+  <div class="error">
+    <div class="error-artwork-container">
+      <picture>
+        <source
+          srcset="/images/errors/small/404.png 576w,
+                  /images/errors/medium/404.png 768w,
+                  /images/errors/original/404.png 1200w">
+        <img src="/images/errors/small/404.png"/>
```
Comment (nilsding) [inline public/422.html]:
```suggestion
        <img src="/images/errors/small/404.png" alt="A raccoon peeking out of a bin, looking confused" />
```

File: public/500.html [hunk 13]
```diff
@@ -1,60 +1,162 @@
...
+    <div class="error">
+    <div class="error-artwork-container">
+      <picture>
+        <source
+          srcset="/images/errors/small/500.png 576w,
+                  /images/errors/medium/500.png 768w,
+                  /images/errors/original/500.png 1200w">
+        <img src="/images/errors/small/500.png"/>
```
Comment (nilsding) [inline public/500.html]:
```suggestion
        <img src="/images/errors/small/500.png" alt="A raccoon smashing a server with a hammer" />
```

**Review by nilsding — APPROVED** [review]

File: public/404.html [hunk 14]
```diff
@@ -1,56 +1,151 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Page not found!!1! (404)</title>
+  <meta charset="utf-8">
+  <title>Page not found</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
```
Comment (raccube) [inline public/404.html:6]:
```suggestion
  <meta name="viewport" content="width=device-width,initial-scale=1">
  <meta name="theme-color" content="#f0edf4">
```

**Review by raccube — CHANGES_REQUESTED** [review]
Just one small thing.

**Review by raccube — APPROVED** [review]

File: public/422.html [hunk 15]
```diff
@@ -1,58 +1,162 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>REJECTED WOAH (422)</title>
+  <meta charset="utf-8">
+  <title>Whoops?</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
```
Comment (raccube) [inline public/422.html:6]:
```suggestion
  <meta name="viewport" content="width=device-width,initial-scale=1">
  <meta name="theme-color" content="#f0edf4">
```

File: public/500.html [hunk 16]
```diff
@@ -1,60 +1,162 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>what is going on (500)</title>
+  <meta charset="utf-8">
+  <title>Internal Server Error</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
```
Comment (raccube) [inline public/500.html:6]:
```suggestion
  <meta name="viewport" content="width=device-width,initial-scale=1">
  <meta name="theme-color" content="#f0edf4">
```

**Review by raccube — CHANGES_REQUESTED** [review]
Whoops, missed these two.

**Review by raccube — APPROVED** [review]

**Pull request merged**